pnpm-lock.yaml

```diff
@@ -43463,6 +43549,8 @@ snapshots:
 
   utils-merge@1.0.1: {}
 
+  uuid@11.1.0: {}
+
   uuid@8.3.2: {}
 
   uuid@9.0.1: {}
```

Comment:
something it still pulling in the old uuid package. I expect this is the type test for old client code pulling in old server code, so maybe this will go away with the next client release.